src/styles/splitter.less

```diff
@@ -48,4 +48,10 @@ as-split {
             .native-focus-outline();
         }
     }
+
+    .as-split-area {
+        &.as-hidden {
+            flex: 0 1 0px !important;
```

Comment:
Looks like this is triggering stylelint:
https://jenkins.swinfra.net/job/SEPG/job/UXAspects/job/UXAspects~UXAspects-2~splitter-update-IE~CI/2/console

I think adding `/* stylelint-disable-line length-zero-no-unit */` should fix it, or something similar. If you could also add a comment linking that GitHub issue to explain why we're doing this, it would help future maintainers.